.markdownlint.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+# Config file for https://github.com/igorshubovych/markdownlint-cli
```

Comment:
```suggestion
# Config file for https://github.com/igorshubovych/markdownlint-cli
```
Is this something to keep or rather some mistaken keyboard hits?

Reply:
I copied this over from the API repo, but you're right, probably not relevant.